mesa/mesaPROTON_OC.py

```diff
@@ -364,13 +371,15 @@ def family_intervene(self):
         5. soc_add_more_friends, a new support member (with a low level of tendency to crime) is added to the friends network
         :return: None
         """
-        kids_to_protect = [agent for agent in self.schedule.agents if agent.age() < 18 and agent.age() >= 12 and agent.father in agent.neighbors.get("parent")]
+        kids_to_protect = [agent for agent in self.schedule.agents if
```

Comment:
Adding breaks to other people's code makes them waste time looking for difference. 

Just don't.